application/java/test/org/openda/costa/CostaOpendaAppTest.java

```diff
@@ -46,29 +45,29 @@ public static void testDummy() {
 		//            the test below is de-activated by renaming it to tst...()
 	}
 
-	public void testOscill_EnKF() throws IOException {
+	public void testOscill_EnKF() {
```

Comment:
Codacy found an issue: [The JUnit 3 test method name 'testOscill_EnKF' doesn't match 'test[A-Z0-9][a-zA-Z0-9]*'](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5102854)